tests/integration/manipulate_page_test.ts

```diff
@@ -0,0 +1,22 @@
+import { HeadlessBrowser } from "../../mod.ts";
+import { assertEquals } from "../../deps.ts";
+
+Deno.test("Manipulate Webpage", async () => {
+  const Sinco = new HeadlessBrowser();
+  await Sinco.build();
+  await Sinco.goTo("https://chromestatus.com");
+
+  const updatedBody = await Sinco.evaluatePage(() => {
+    // deno-lint-ignore no-undef
+    const prevBody = document.body.children.length;
+    // deno-lint-ignore no-undef
+    const newEl = document.createElement("p");
+    // deno-lint-ignore no-undef
+    document.body.appendChild(newEl);
+    // deno-lint-ignore no-undef
+    return prevBody === document.body.children.length - 1;
```

Comment:
Shouldn't the. `document.body.children.length` now be one more than `prevBody`? 

Eg

```
const prevBody = ... // 2 children
document.append(p)
const currentBody. = ... // 3 children
```

So that eval should return false right?

Reply:
this is `true`. `prevBody` is one less than `document.body.children` because we captured that val _before_ the `appendChild` call

so `prevBody === document.body.children.length` will return `false` but `prevBody === document.body.children.length - 1` is accurate

Reply:
Ohhh i see!! Woooooowwwwwwwwwwwwwwwwwww and i call myself a developer